admin/javascript/src/components/form-action/styles.scss

```diff
@@ -0,0 +1,124 @@
+// General buttons
+.btn {
+	height: 32px;
+	margin-right: 1rem;
+	position: relative;
+}
+
+// Button icons
+.btn[class*="font-icon-"]::before {
+	font-size: 16px;
+	position: relative;
+	top: 3px;
+	margin-right: 6px;
+	line-height: 13px;
+}
+
+.no-text[class*="font-icon-"]::before {
+	margin-right: 0;
+}
+
+.btn-group {
+	margin-right: 1rem;
+
+	.btn {
+		margin-right: 0;
+	}
+
+	.btn-success {
+		border-left: 1px solid darken($btn-success-bg, 6%);
+
+		&:first-child {
+			border-left: none;
+		}
+	}
+}
+
+// SVG loading icon
+.btn__loading-icon {
+	float: left;
+	margin: 0 4px 0 0;
+	height: 20px;
+	position: absolute;
+	left: 50%;
+	top: $btn-padding-y;
+	transform: translate(-50%);
+
+	svg {
+		width: 24px;
```

Comment:
Presume you've checked in IE? http://caniuse.com/#feat=svg says IE up to 11 doesn't scale properly.